src/styles/main.scss

```diff
@@ -1,18 +1,19 @@
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
-@import "variables";
-@import "mixins";
-@import "../components/**/*.scss"; // find scss files in whole components directory tree
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap');
+@import 'mixins';
+@import '../components/**/*.scss'; // find scss files in whole components directory tree
 
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   font-display: swap;
-  position: relative;
 }
 
-html, body {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Roboto Light", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+html,
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Roboto Light', 'Oxygen',
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/manual/matfilipowicz/BooGi/pullRequest?prid=5801182)